src/Cache/Adapter/Memcached.php

```diff
@@ -74,4 +74,14 @@ public function flush(): bool
     {
         return $this->memcached->flush();
     }
+
+    /**
+     * @return bool
+     */
+    public function ping(): bool
+    {
+        $this->save('utopia-ping', 'pong');
```

Comment:
We should document this at least in the README that there is now a reserved keyword `utopia-ping`. Maybe even throw `ReservedKeywordException` when developer tries to use it.

Reply:
Fixed so we don't need reserved keywords at all.